docs/networking/deep-dive.md

```diff
@@ -0,0 +1,112 @@
+---
+sidebar_position: 3
+sidebar_label: Deep Dive
+title: ""
+keywords:
+- Harvester
+- Networking
+- Topology
+---
+# Harvester Network Deep Dive
+
+The network topology below reveals how we implement the Harvester network.
+
+![](/img/v1.1/networking/topology.png)
+
+The diagram contains [the built-in cluster network mgmt](./clusternetwork.md/#built-in-cluster-network) and a [custom cluster network](./clusternetwork.md/#custom-cluster-network) called `oob`.
+
+As shown above, the Harvester network primarily focuses on layer 2. We leverage Linux network devices to construct traffic paths for the communication between VM to VM, VM to host, and VM to external network devices.
```

Comment:
`layer 2` -> `OSI model layer 2`
`Linux network devices` -> `Linux network devices and protocols`